src/main/java/de/wejago/hichi2influx/config/MqttClientConfig.java

```diff
@@ -21,6 +22,7 @@ public class MqttClientConfig {
 
     private final MqttProperties mqttProperties;
 
+    @Bean
     public IMqttClient getMqttClient() {
```

Comment:
Any concerns if we rename it to MqttClient? I dont like the I at the begining :)

Reply:
With the interface we are more flexible here.
I already thought about the possibility to create an auto-re-connecting MQTT client, that is a wrapper around the default MQTT client.
Because we need some robust functionallity in the future that tries to reconnect in case of problems/disconnect.